website/utils/url_utils.ts

```diff
@@ -0,0 +1,185 @@
+import FirebaseRequest, { DatasetMetaData } from "../../public/firebase";
+
+import { AppProps, GlobalViewerSettings } from "../../src/aics-image-viewer/components/App/types";
+import { ViewMode } from "../../src/aics-image-viewer/shared/enums";
+import { ViewerChannelSettings } from "../../src/aics-image-viewer/shared/utils/viewerChannelSettings";
+
+const paramKeys = ["mask", "ch", "luts", "colors", "url", "file", "dataset", "id", "view"];
```

Comment:
an interesting todo is to add comments describing how these are parsed.  In particular, the lut setting has some special shorthand encoding using p for percentile and m for median. For later..

Reply:
Will likely do this when #196 is implemented 👍